src/services/projects.js

```diff
@@ -352,21 +361,38 @@ module.exports = class ProjectsHelper {
 	 * @method
 	 * @name details
 	 * @param {String} projectId - Project id
+	 * @param {String} orgCode - Project id
+	 * @param {String} tenantCode - Project id
 	 * @returns {JSON} - Project data.
 	 */
 
-	static async details(projectId) {
+	static async details(projectId, orgCode, tenantCode, userId, commentsOptions = {}) {
```

Comment:
@adithyadinesh0412 Can u check if mandating the new key will break any other function ?